composer.json

```diff
@@ -20,8 +20,11 @@
             "role": "Developer"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
You probably don't need this.

Reply:
I needed it because of https://github.com/mautic/mautic/issues/10651
Got this solution from `GrapesJsBuilderBundle` 

Added `prefer-stable` so it uses less `dev` versions